Userland/Services/LookupServer/DNSPacket.h

```diff
@@ -38,8 +29,12 @@ class DNSPacket {
 
     bool is_query() const { return !m_query_or_response; }
     bool is_response() const { return m_query_or_response; }
+    bool is_authoritative_answer() const { return m_authoritative_answer; }
+    bool is_recursion_desired() const { return m_recursion_desired; }
     void set_is_query() { m_query_or_response = false; }
     void set_is_response() { m_query_or_response = true; }
+    void set_is_authoritative_answer() { m_authoritative_answer = true; }
+    void set_is_recursion_desired() { m_recursion_desired = true; }
```

Comment:
These two setters should probably take booleans?
(Or have a sister method that sets the opposite value, like `set_is_query()` & `set_is_response()` above.)

Reply:
Yeah, they should. I've changed that now to match the setter and getter names from `DNSPacketHeader`.